src/modules/Kdump.rb

```diff
@@ -966,7 +972,7 @@ def Import(settings)
     #
     # @return [Boolean] is supported
     def fadump_supported?
```

Comment:
Well, it's used from the clients and from the UI and I though that maybe it was worth keeping it as a way to "hide" implementation details to the layers on top of the Kdump YCP-style module. But I guess I could also simply substitute all the calls with `Kdump.system.supports_fadump?`